target/linux/ath79/dts/qca9563_dlink_dir-859-a1.dts

```diff
@@ -41,112 +41,3 @@
 		};
 	};
 
-	keys {
```

Comment:
Please  add `#include "qca9563_dlink_dir-8x9-a1.dtsi"` into this file too. 

Please also compile test all the boards you modified.